docs/book/src/clusterctl/commands/commands.md

```diff
@@ -1,19 +1,22 @@
+| [`clusterctl alpha rollout`](alpha-rollout.md)                               | Manages the rollout of Cluster API resources. For example: MachineDeployments.                             |
+| [`clusterctl alpha topology plan`](alpha-topology-plan.md)                   | Describes the changes to a cluster topology for a given input.                                             |
+| [`clusterctl backup`](additional-commands.md#clusterctl-backup)              | Backup Cluster API objects and all their dependencies from a management cluster.                           |
+| [`clusterctl completion`](completion.md)                                     | Output shell completion code for the specified shell (bash or zsh).                                        |
+| [`clusterctl config`](additional-commands.md#clusterctl-config-repositories) | Display clusterctl configuration.                                                                          |
+| [`clusterctl delete`](delete.md)                                             | Delete one or more providers from the management cluster.                                                  |
+| [`clusterctl describe cluster`](describe-cluster.md)                         | Describe workload clusters.                                                                                |
+| [`clusterctl generate cluster`](generate-cluster.md)                         | Generate templates for creating workload clusters.                                                         |
+| [`clusterctl generate provider`](generate-provider.md)                       | Generate templates for provider components.                                                                |
+| [`clusterctl generate yaml`](generate-yaml.md)                               | Process yaml using clusterctl's yaml processor.                                                            |
+| [`clusterctl get kubeconfig`](get-kubeconfig.md)                             | Gets the kubeconfig file for accessing a workload cluster.                                                 |
+| [`clusterctl help`](additional-commands.md#clusterctl-help)                  | Help about any command.                                                                                    |
+| [`clusterctl init`](init.md)                                                 | Initialize a management cluster.                                                                           |
+| [`clusterctl move`](move.md)                                                 | Move Cluster API objects and all their dependencies between management clusters.                           |
+| [`clusterctl restore`](additional-commands.md#clusterctl-restore)            | Restore Cluster API objects from file by glob. Object files are searched in config directory.              |
```

Comment:
```suggestion
| [`clusterctl restore`](additional-commands.md#clusterctl-restore)            | Restore Cluster API objects from file by glob.              |
```
OR
```suggestion
| [`clusterctl restore`](additional-commands.md#clusterctl-restore)            | Restore Cluster API objects from file by glob.  Object files are searched in the config directory or in the provided directory.            |
```

Reply:
Thank you for the suggestions! I took the first to keep it shorter.